group_vars/matrix_servers

```diff
@@ -622,6 +622,41 @@ matrix_mx_puppet_steam_database_password: "{{ matrix_synapse_macaroon_secret_key
 #
 ######################################################################
 
+######################################################################
+#
+# matrix-bridge-mx-puppet-groupme
+#
+######################################################################
+
+# We don't enable bridges by default.
+matrix_mx_puppet_groupme_enabled: false
+
+matrix_mx_puppet_groupme_container_image_self_build: "{{ matrix_architecture != 'amd64'}}"
+
+matrix_mx_puppet_groupme_systemd_required_services_list: |
+  {{
+    ['docker.service']
+    +
+    (['matrix-synapse.service'] if matrix_synapse_enabled else [])
+    +
+    (['matrix-postgres.service'] if matrix_postgres_enabled else [])
+  }}
+
+matrix_mx_puppet_groupme_appservice_token: "{{ matrix_synapse_macaroon_secret_key | password_hash('sha512', 'mxste.as.tok') | to_uuid }}"
+
+matrix_mx_puppet_groupme_homeserver_token: "{{ matrix_synapse_macaroon_secret_key | password_hash('sha512', 'mxste.hs.tok') | to_uuid }}"
```

Comment:
These should be changed to something unique. Right now it conflicts with the Steam bridge.

Perhaps `mxgro.as.tok` and `mxgro.hs.tok`.

Reply:
Missed that one, thanks.